pkg/template/template_test.go

```diff
@@ -0,0 +1,53 @@
+package template_test
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/template"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testTemplates/*
+var EFS embed.FS
+
+//go:embed testTemplates/host/*
+var hostFS embed.FS
+
+func TestLoadObjectsFromEmbedFS(t *testing.T) {
```

Comment:
could you please add test-cases that validate the exact content of the objects that are in the templates?
These are dummy templates with values that won't change often (compared to what we have in host/member-operator repos), so we can afford hard-coding the expected values for the given kinds in the tests.

Reply:
Makes sense, please take a look at https://github.com/codeready-toolchain/toolchain-common/pull/365/commits/879f02df98cf7ae4da15c256c8551e1de847c026 and let me know if this could be enough to cover your suggestion.